src/Native/Runtime/gcrhenv.cpp

```diff
@@ -1148,6 +1167,180 @@ Thread* GCToEEInterface::CreateBackgroundThread(GCBackgroundThreadFunction threa
     return threadStubArgs.m_pThread;
 }
 
+void GCToEEInterface::DiagGCStart(int gen, bool isInduced)
+{
+#ifdef GC_PROFILING
```

Comment:
I do not think that `GC_PROFILING` is ever defined in CoreRT/ProjectN. These methods should have empty implementations; or have just the event trace stuff in them.